runtime/service.go

```diff
@@ -574,7 +584,6 @@ func (s *service) StopVM(requestCtx context.Context, request *proto.StopVMReques
 		timeout = time.Duration(request.TimeoutSeconds) * time.Second
 	}
 
-	defer s.shimCancel()
```

Comment:
Do we also want to remove the s.shimCancel() in the case of CreateVM errors and move that to a cleanup call?

Reply:
Yes. We should reduce the number of places where we call shimCancel.